README.md

```diff
@@ -6,7 +6,106 @@ The Microsoft Graph Communications API allows developers to programmatically int
 
 - View the **[changelog](changelog.md)**
 - **[Review the documentation](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/)** to understand the concepts behind using our SDK (which is also used by the samples).
-- Get started quickly with our **[samples](https://github.com/microsoftgraph/microsoft-graph-comms-samples/tree/master/Samples)**.
+
+## Prerequisites
+
+* Visual Studio. You can download the community version [here](http://www.visualstudio.com) for free.
+* Mirosoft Azure Subscription (If you do not already have a subscription, you can register for a <a href="https://azure.microsoft.com/en-us/free/" target="_blank">free account</a>)
```

Comment:
Why are we using an `a` tag here, not using md link syntax?

Reply:
nice pickup